### PR TITLE
feat: Create Suggestion Input component

### PR DESCRIPTION
**Pull request by aaschlote**

### CODE REVIEW

File: app/components/inputs/SuggestionInput.tsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+}
+
+const filterOption = (option: DataListOptions, inputValue: string) => {
+  if (inputValue.length <= 2) {
+    return false;
+  }
+  return option.label.toLowerCase().includes(inputValue.toLocaleLowerCase());
+};
+
+function getDescriptioByValue(
```
Comment (pgurusinga) [inline app/components/inputs/SuggestionInput.tsx:70]: a type 
```suggestion
function getDescriptionByValue(
```

File: app/components/inputs/SuggestionInput.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+          ? `${airport.airport} (${airport.iata})`
+          : `${airport.city} ${airport.airport} (${airport.iata})`,
+        subDescription: `${airport.city}, ${airport.country}`,
+      }));
+  }
+  return [];
+}
+
+const filterOption = (option: DataListOptions, inputValue: string) => {
+  if (inputValue.length <= 2) {
```
Comment (pgurusinga) [inline app/components/inputs/SuggestionInput.tsx:64]: I think this should be 3, based on the requirement in the ticket (`Suggesting (once the 3 first letters are typed`)

File: app/components/inputs/SuggestionInput.tsx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+        filterOption={filterOption}
+        defaultValue={currentItemValue}
+        placeholder={placeholder}
+        instanceId={name}
+        formatOptionLabel={formatOptionLabel}
+        onChange={getInputProps().onChange}
+        onBlur={getInputProps().onBlur}
+        noOptionsMessage={({ inputValue }) =>
+          inputValue.length > 2
+            ? "Leider konnten wir kein passenden Flughafen finden. Bitte prüffen Sie ihre Angabe"
```
Comment (pgurusinga) [inline app/components/inputs/SuggestionInput.tsx:200]: hmm probably we need to have the string in the Strapi

File: app/components/inputs/SuggestionInput.tsx (hunk 2 above)
Comment (pgurusinga) [reply app/components/inputs/SuggestionInput.tsx:64]: ah no this is already correct! 🙏

File: app/components/inputs/SuggestionInput.tsx [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+        aria-describedby={error && errorId}
+        aria-errormessage={error && errorId}
+        id={name}
+        name={name}
+        filterOption={filterOption}
+        defaultValue={currentItemValue}
+        placeholder={placeholder}
+        instanceId={name}
+        formatOptionLabel={formatOptionLabel}
+        onChange={getInputProps().onChange}
```
[the remark this answers is not in the document]
Comment (aaschlote) [reply app/components/inputs/SuggestionInput.tsx:196]: I didn't get your point here, let's synch late about it :)

File: app/components/inputs/SuggestionInput.tsx (hunk 1 above)
Comment (aaschlote) [reply app/components/inputs/SuggestionInput.tsx]: Thanks for pointing out all my typos 😄

File: app/components/inputs/SuggestionInput.tsx (hunk 3 above)
Comment (aaschlote) [reply app/components/inputs/SuggestionInput.tsx]: Done

**Review by pgurusinga — APPROVED** [review]

**Pull request merged**